kafka/librdkafka_vendor/bundle-import.sh

```diff
@@ -32,6 +32,11 @@ parse_dynlibs() {
             libs="${libs} $n"
         fi
     done
+    for n in $(grep ^Libs.private: $pc); do
```

Comment:
Why is Libs.private needed?
rdkafka-static.pc defines Libs:, not Libs-private (there are issues with that which I can't remember now)

Reply:
This is where the `-lws2_32 -lsecur32 -lcrypt32` flags get defined for the MinGW build. Based on https://people.freedesktop.org/~dbn/pkg-config-guide.html, this appears to be correct behavior. Will this cause issues for one of the other static bundles?

Reply:
I have a weak recollection that `pkg-config --libs --static ..` misbehaves when it comes to Libs.private, something about that it may pick up a dynamic library instead of a static, depending on path circumstances, or something with secondary dependencies. I can't really remember the exact details, but I do remember there being a strong reason for splitting out rdkafka .pc files into a static and non-static variant, and I'm not comfortable with ignoring that here, even though I don't have the full answer to why.

Reply:
I can appreciate that. I see two approaches:
    1. Modify the librdkafka CMake files to only use Libs for the static builds
    2. Modify bundle_import.sh to manually pop `-lws2_32 -lsecur32 -lcrypt32` for `build_windows.go`
Option #1 seems far more attractive to me. Agree?

Reply:
Also, I found a description of the misbehavior you were talking about in one of the links I sent
https://autotools.io/pkgconfig/dependencies.html
(Emphasis mine)
>2.1. Static linking
*When using the private definitions for dependencies, the behaviour of pkg-config will change depending on whether it's targeting dynamic or static linking*. In the former case, private dependencies will not be listed in the output, as they are not exposed to the consumer directly, and shared libraries will handle them.
>
>Contrarily, *when the link is static, it'll expand (recursively) all the private dependencies* so that all the libraries are brought in. This is once again needed because of the way UNIX link editors work, as archives do not list their own dependencies directly. While libtool actually provides a wrapper around these archives, they are getting less and less common, so *relying on them is not a good idea*.
>
>Unfortunately, at least as of February 2013, there is no easy way to tell at once to an Autotools-based build system that you intend to link statically; this means that the most common way to signal this to pkg-config becomes the command ./configure LDFLAGS="-static" PKG_CONFIG="pkg-config --static".

Reply:
PR opened for option 1
https://github.com/edenhill/librdkafka/pull/3354